addons/account_peppol/models/account_edi_proxy_user.py

```diff
@@ -192,17 +195,16 @@ def _peppol_get_new_documents(self):
                             "mimetype": "application/xml",
                         }
                     )
-                    if edi_user._peppol_import_invoice(attachment, None, content["state"], uuid):
-                        # Only acknowledge when we saved the document somewhere
-                        proxy_acks.append(uuid)
+                    created_moves |= edi_user._peppol_import_invoice(attachment, None, content["state"], uuid, journal)
 
                 if not modules.module.current_test:
                     self.env.cr.commit()
-                if proxy_acks:
+                if created_moves:
                     edi_user._call_peppol_proxy(
                         "/api/peppol/1/ack",
-                        params={'message_uuids': proxy_acks},
+                        params={'message_uuids': list(created_moves.mapped('peppol_message_uuid'))},
                     )
+                    journal._postprocess_received_einvoices(created_moves)
```

Comment:
This method is unclear. What about
```suggestion
                    journal._notify_einvoices_received(created_moves)
```